### PR TITLE
Create readme

### PR DESCRIPTION
**Pull request by falcao-g**
This pr creates a initial readme, listing basically what this project does and what it uses

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<h1 align='center'> PS Back-end Mega 2023 </h1>
+
+<section align='center'>
+<img alt="Top language" src="https://img.shields.io/github/languages/top/falcao-g/processo-mega">
+<img alt="Version express" src="https://img.shields.io/badge/express--red.svg?logo=express">
```
Comment (LKodex) [inline README.md]: It would be nice if you add the version on the badges
like this:
/express-^4.18.2-red.svg?logo=express

**Review by LKodex — APPROVED** [review]
Nice work, the readme looks great

File: README.md (hunk 1 above)
Comment (falcao-g) [reply README.md]: I added this and was unsure of what to add in the postgres one so i added the version of the pg package

Comment (LKodex) [reply README.md]: Let's use the latest postgres version which is 15

Comment (falcao-g) [reply README.md]: done

**Review by LKodex — APPROVED** [review]

**Pull request merged**